docs/Reference/Cakeshop/Overview.md

```diff
@@ -0,0 +1,79 @@
+# Getting started with Cakeshop
+
+## Download
+
+Binary packages are available on the [GitHub releases page](https://github.com/ConsenSys/cakeshop/releases).
+
+## Configuration
+
+Cakeshop is a Spring Boot application, so you may place an `application.properties` file in the working directory to override any default configuration values. For more info, see the [configuration page](Configuration.md).
+
+## Running via GoQuorum Wizard
+
+The easiest way to use Cakeshop is to generate a GoQuorum network with [GoQuorum Wizard](../../HowTo/GetStarted/Wizard/GettingStarted.md) and choose to deploy Cakeshop alongside the network.
+
+## Requirements
+
+* Java 11+
+* Nodejs (if the Nodejs binary on your machine isn't called 'node', see [here](Configuration.md#cakeshop-internals))
+
+## Running
+
+* Download WAR file
+* Run `java -jar cakeshop.war`
+* Navigate to [http://localhost:8080/](http://localhost:8080/)
+
+## Running via Docker
+
+Simple example of running via docker on port 8080:
+
+```sh
+docker run -p 8080:8080 quorumengineering/cakeshop
+```
+
+Then access the UI at [http://localhost:8080/](http://localhost:8080/)
+
+## Docker Customizations
+
+You can add some extra flags to the run command to further customize Cakeshop.
+
+Here is an example where you mount `./data` as a data volume for the container to use:
+
+```sh
+mkdir data
+docker run -p 8080:8080 -v "$PWD/data":/opt/cakeshop/data quorumengineering/cakeshop
+```
+
+An example providing an initial nodes.json in the data directory and configuring it to be used:
+
+```sh
+# makes sure you have nodes.json at $PWD/data/nodes.json
+docker run -p 8080:8080 -v "$PWD/data":/opt/cakeshop/data \
+    -e JAVA_OPTS="-Dcakeshop.initialnodes=/opt/cakeshop/data/nodes.json" \
+    quorumengineering/cakeshop
+```
+
+## Migrating from Cakeshop v0.11.0
+
+The following big changes were made in v0.12.0:
+
+1. Simplification of config file to better follow Spring Boot standards.
+1. Moved Contract Registry from being stored in a combination of a smart contract and the database to being in the database only.
+1. Elimination of Cakeshop's managed node in favor of only attaching to existing nodes.
+1. Simplified DB configuration by using Spring Data.
+
+To ensure easy transition, Cakeshop will still look in the locations where v0.11.0 commonly stored the config file. But (1) allows you to now place your config in the folder where you run Cakeshop, or specify a different location using standard spring boot flags, for easier customization.
```

Comment:
maybe adding these details as sub items in the previous list. It's to prevent the need to go back and forth. Having a list sub item makes it clear that it's detail about the parent item.